packages/playwright/src/isomorphic/teleReceiver.ts

```diff
@@ -14,9 +14,8 @@
  * limitations under the License.
  */
 
-import type { Metadata } from '../../types/test';
+import type { Metadata, TestAnnotation } from '../../types/test';
```

Comment:
Please also look at `_updateLocation` inside `merge.ts` - we would need to patch these locations similarly to all others.

Reply:
Good catch. This was difficult to solve for. We should improve this area of the code.